libs/feature/record/src/lib/data-view-table/data-view-table.component.ts

```diff
@@ -84,9 +84,9 @@ export class DataViewTableComponent {
         .getGeoJsonDownloadUrlFromWfs(link.url, link.name)
         .pipe(
           switchMap((url) =>
-            this.dataService.readGeoJsonDataset(url).pipe(
-              map((featureCollection) =>
-                featureCollection.features.map((f) => ({
+            this.dataService.readDataset(url, 'geojson').pipe(
+              map((features) =>
```

Comment:
Slightly confusing why you introduce a type `DataItem = Feature` and call the arg `features`.